crypto/ec/ec_curve.c

```diff
@@ -3246,3 +3248,158 @@ int EC_curve_nist2nid(const char *name)
     }
     return NID_undef;
 }
+
+static inline
+int ec_group_get_curve(const EC_GROUP *group, BIGNUM *p, BIGNUM *a,
+                       BIGNUM *b, BN_CTX *ctx)
+{
+    int field_nid;
+
+    field_nid = EC_METHOD_get_field_type(EC_GROUP_method_of(group));
+
+#ifndef OPENSSL_NO_EC2M
+    if (field_nid == NID_X9_62_characteristic_two_field) {
+        return EC_GROUP_get_curve_GF2m(group, p, a, b, ctx);
+    } else
+#endif /* !def(OPENSSL_NO_EC2M) */
+    if (field_nid == NID_X9_62_prime_field) {
+        return EC_GROUP_get_curve_GFp(group, p, a, b, ctx);
+    } else {
+        /* this should never happen */
+        return 0;
+    }
+}
+
+static inline
+int ec_point_get_affine_coordinates(const EC_GROUP *group,
+                                    const EC_POINT *point, BIGNUM *x,
+                                    BIGNUM *y, BN_CTX *ctx)
+{
+    int field_nid;
+
+    field_nid = EC_METHOD_get_field_type(EC_GROUP_method_of(group));
+
+#ifndef OPENSSL_NO_EC2M
+    if (field_nid == NID_X9_62_characteristic_two_field) {
+        return EC_POINT_get_affine_coordinates_GFp(group, point, x, y, ctx);
+    } else
+#endif /* !def(OPENSSL_NO_EC2M) */
+    if (field_nid == NID_X9_62_prime_field) {
+        return EC_POINT_get_affine_coordinates_GF2m(group, point, x, y, ctx);
+    } else {
+        /* this should never happen */
+        return 0;
+    }
+}
```

Comment:
While we are addressing the `static inline` functions, I was thinking that maybe these 2 wrappers could actually go in `ec_lib.c` and available internally through `ec_lcl.h`

@mattcaswell what do you think about it?

Reply:
Regarding `inline` in `1.0.2` I realized we avoid `static inline` altogether in the crossplatform C code, so I will push a fixup commit to drop `inline` for now.

(I will wait for Appveyor to finish the queued jobs for this PR as it seems it has been blocking the pipeline with the frequent fixups so far)

Reply:
>  what do you think about it?

I'd be ok with that.

Reply:
> > what do you think about it?
> 
> I'd be ok with that.

See 06dc1c4bc60b4f79b1a00fe277791a261b04437e